examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -1681,3 +1785,25 @@ pub extern "C" fn channel_loser(_handle: u64) {
     // On Node exit (when this function returns), the Runtime should free those
     // channels.
 }
+
+fn new_channel_chain(nest_count: u32) -> Result<oak::ReadHandle, Box<dyn std::error::Error>> {
```

Comment:
Could you add a comment, and perhaps even a little ASCII-art diagram, if you feel like it?

Reply:
Added comment but no ASCII art.